File: problemas/1000/1012.cs
using System;

namespace uri1012
{
    internal static class Program
    {
        private static void Main()
        {
            string[] entrada = Console.ReadLine().Split(' ');
            double a = double.Parse(entrada[0]);
            double b = double.Parse(entrada[1]);
            double c = double.Parse(entrada[2]);
            Console.WriteLine($"TRIANGULO: {(a * c / 2.0):F3}");
            Console.WriteLine($"CIRCULO: {(3.14159 * c * c):F3}");
            Console.WriteLine($"TRAPEZIO: {((a + b) * c / 2.0):F3}");
            Console.WriteLine($"QUADRADO: {(b * b):F3}");
            Console.WriteLine($"RETANGULO: {(a * b):F3}");
        }
    }
}

File: problemas/1000/1012.php
<?php

$entrada = explode(' ', readline());
$a = $entrada[0];
$b = $entrada[1];
$c = $entrada[2];
echo 'TRIANGULO: ' . number_format($a * $c / 2.0, 3, '.', '') . "\n";
echo 'CIRCULO: ' . number_format(3.14159 * $c * $c, 3, '.', '') . "\n";
echo 'TRAPEZIO: ' . number_format(($a + $b) * $c / 2.0, 3, '.', '') . "\n";
echo 'QUADRADO: ' . number_format($b * $b, 3, '.', '') . "\n";
echo 'RETANGULO: ' . number_format($a * $b, 3, '.', '') . "\n";